packages/components/src/templates/next/components/internal/ChildrenPages/ChildrenPages.tsx

```diff
@@ -1,13 +1,135 @@
+  groupFocusVisibleHighlight,
+  isExternalUrl,
+} from "~/utils"
+import { ImageClient, ImageClientProps, InfoCards } from "../../complex"
+import { ComponentContent } from "../customCssClass"
+
+const ChildpageImage = ({
+  src,
+  alt,
+  assetsBaseUrl,
+  className,
+}: Pick<ImageClientProps, "className" | "assetsBaseUrl" | "src" | "alt">) => {
+  const imgSrc =
+    isExternalUrl(src) || assetsBaseUrl === undefined
+      ? src
+      : `${assetsBaseUrl}${src}`
+
+  return (
+    <ImageClient
+      assetsBaseUrl={assetsBaseUrl}
+      width="100%"
+      className={className}
+      alt={alt}
+      src={imgSrc}
+    />
+  )
+}
+
+interface Childpage {
+  title: string
+  url: string
+  description: string
+  image?: IsomerSitemap["image"]
+}
+
+interface RowLayoutProps {
+  childpages: Childpage[]
+  showSummary: boolean
+  showThumbnail: boolean
+  assetsBaseUrl?: string
+  fallback: Required<NonNullable<IsomerSitemap["image"]>>
+}
+
+const createRowStyles = tv({
+  slots: {
+    container: `${ComponentContent} grid grid-cols-3 md:grid-cols-6 md:gap-x-10 lg:grid-cols-12`,
+    image:
+      "aspect-[3/2] object-cover max-md:col-span-full max-md:row-span-1 md:col-span-2 lg:col-span-3",
+    textContainer: "max-md:col-span-full max-md:row-span-1",
+    contentContainer:
+      "max-md:grid-rows-[1fr fit-content] grid grid-cols-subgrid rounded-sm border-t border-t-base-divider-medium py-5 last:border-b last:border-t-base-divider-medium max-md:col-span-full max-md:gap-y-5 md:col-span-6 lg:col-span-12",
+    title: [
+      groupFocusVisibleHighlight(),
+      "prose-title-md-medium text-base-content-strong",
+    ],
+    description: "gray prose-body-base text-gray-700",
+  },
+  variants: {
+    layout: {
+      default: {},
+    },
+    hasThumbnail: {
+      true: { textContainer: "md:col-span-4 lg:col-span-9" },
+      false: { textContainer: "md:col-span-6 lg:col-span-12" },
+    },
+  },
+
+  defaultVariants: {
+    layout: "default",
+  },
+})
+
+const RowLayout = ({
+  childpages,
+  showSummary,
+  showThumbnail,
+  assetsBaseUrl,
+  fallback,
+}: RowLayoutProps): JSX.Element => {
+  const styles = createRowStyles()
+
+  return (
+    <div className={styles.container()}>
+      {childpages.map(({ title, description, url, image }) => {
+        const renderedImage = image?.src ? image : fallback
+
+        return (
+          <div
+            className={styles.contentContainer({
+              hasThumbnail: !!showThumbnail,
+            })}
+          >
+            {showThumbnail && (
+              <ChildpageImage
+                assetsBaseUrl={assetsBaseUrl}
+                {...renderedImage}
+                className={styles.image()}
+              />
+            )}
+            <div
+              className={styles.textContainer({
+                hasThumbnail: !!showThumbnail,
+              })}
+            >
+              <a href={url} className={styles.title()}>
+                {title}
+              </a>
```

Comment:
use `packages/components/src/templates/next/components/internal/Link/Link.tsx` so that there will be nextjs prefetching?

Reply:
ah no - this is actually **wrong**! we should instead use `LinkComponent` (:monkas:)